tests/foreman/ui/test_user.py

```diff
@@ -258,6 +258,52 @@ def test_positive_create_with_all_roles(self):
                 self.assertIsNotNone(self.user.wait_until_element(
                     (strategy, value % role)))
 
+    @tier2
+    def test_positive_create_with_multiple_roles_and_login(self):
+        """Create User with multiple roles and re-login into application
+        using that user
+
+        @id: b1d96e56-42e0-4f4b-8f38-98fb72be5c64
+
+        @Assert: User is created successfully and can be used to login into
+        application without raising any errors. Also user can review his own
+        profile data.
+
+        @BZ: 1392513
```

Comment:
That is right + I did that intentionally as that test is directly targeted on BZ coverage from the table provided by Jake, so gonna do that all the time when work with it